qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1517A.py

```diff
@@ -49,6 +79,42 @@ def __init__(self, parent: 'KeysightB1500', name: Optional[str], slot_nr,
             get_cmd=self._get_current
         )
 
+        self.add_parameter(
+            name="time_axis",
+            get_cmd=self._get_time_axis,
+            vals=Arrays(shape=(self._get_sampling_number,))
+        )
+
+        self.add_parameter(
+            name="sampling_measurement",
```

Comment:
i'd suggest to call this parameter `sampling_measurement_trace` or smth, not just `sampling_measurement` as it is too abstract and doesn't really convey that it's a parameter.

Reply:
Yes, it is too abstract. The name is changed according to your suggestion.